### PR TITLE
Add gluster_volume_option custom resource

### PR DESCRIPTION
**Pull request by chewi**

### CODE REVIEW

**Comment by chewi** [comment]
I've got work piling on top of me but I understand you want to maintain quality so I'll try to find the time. I've also got geo-replication/mountbroker stuff on the way.

**Comment by chewi** [comment]
Misunderstood you earlier, thought you wanted me to write some specs. I've never actually used ChefSpec (but I should…) so I've not tested these but seems simple enough. I would have thought this could automated but what do I know? :grin:

**Comment by shortdudey123** [comment]
Take a look at `libraries/matchers.rb` and it should be fairly self explanatory to add them to that file

def name is `<action>_<resource>`
and parameters are `(<resource>, <action>, resource_name)`

**Comment by chewi** [comment]
Yep, I meant I've already done it, please see the amended commit.

**Comment by chewi** [comment]
Sorry, just realised I messed up the README ordering slightly. I'll fix it tomorrow as it's late now.

File: libraries/matchers.rb [hunk 1]
```diff
@@ -14,4 +14,12 @@ def enable_gluster_mount(resource_name)
   def disable_gluster_mount(resource_name)
     ChefSpec::Matchers::ResourceMatcher.new(:gluster_mount, :disable, resource_name)
   end
+
+  def set_gluster_volume_option(resource_name) # rubocop:disable Style/AccessorMethodName
```
Comment (shortdudey123) [inline libraries/matchers.rb]: Can you actually put the rubocop rule in `.rubocop.yml` as an excludes for this file?

**Comment by chewi** [comment]
Okay, all sorted.

**Comment by shortdudey123** [comment]
Awesome work!

**Pull request merged**